### DISCUSSION TITLE
How to run mutation only on diff file change on Gitlab CI

### DISCUSSION THREAD

**Discussion by lyjidapa** [stryker-mutator/stryker-net#2003] [Q&A] [answered]
I'm working on .Net project and wondering is there any command to run only diff file change between MR and Master.

**Comment by rouke-broersma** [comment, marked as answer]
https://stryker-mutator.io/docs/stryker-net/configuration#since-flag-committish

https://stryker-mutator.io/docs/stryker-net/stryker-in-pipeline#configuring-baseline-in-pull-requests
